src/olympia/migrations/1018-drop-title-from-rating.sql

```diff
@@ -0,0 +1,14 @@
+/* Drop rating title */
+
+SET @CONSTRAINT_NAME := (SELECT CONSTRAINT_NAME FROM information_schema.key_column_usage WHERE TABLE_SCHEMA=(SELECT DATABASE()) AND TABLE_NAME="reviews" and COLUMN_NAME="title" AND REFERENCED_TABLE_NAME = "translations");
+
+SET @QUERY = CONCAT('ALTER TABLE reviews DROP FOREIGN KEY ', @constraint_name, ';');
+PREPARE stmt FROM @QUERY;
+EXECUTE stmt;
+DEALLOCATE PREPARE stmt;
+
+/* Ignore the translations because our databases are a bag of crazy. */
```

Comment:
:thinking: although that actually plays right into exactly that fact. Maybe it's easier to clean-up with a one-off python management command that deletes all instances that are related to `reviews.title`? But then again… we already removed that from the model so that won't work. Mhmm… Just thinking loud :laughing:

Reply:
IMHO the best way to do this is to make a crazy SQL query that deletes orphaned translations by looking at the mysql schema tables to find all possible fks and finding translations with an `id` that are not referenced by any table.

Reply:
Tbh this was minimally copied from previous times we've dropped and/or flattened translated fields.  It's _possible_ to delete translations with an `id` matching the value of the `title` column but I think we were just too afraid of deleting something by accident.

My opinion is still to replace all the translation nonsense in the database with something better and lose the orphaned translations along the way :)